vignettes/mmrm_review_methods.Rmd

```diff
@@ -0,0 +1,305 @@
+output:
+  rmarkdown::html_document:
+          theme: "spacelab"
+          highlight: "kate"
+          toc: true
+          toc_float: true
+          toc_depth: 2
+vignette: |
+  %\VignetteEncoding{UTF-8}
+  %\VignetteIndexEntry{Implementations of MMRM: a comparison of what's available}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options:
+  chunk_output_type: console
+  markdown:
+    wrap: 72
+---
+
+# Introduction
+
+In this vignette we will show comparisons between `mmrm`, `SAS PROC MIXED`, `nlme::gls`, `lme4::lmer`, and `glmmTMB` implementations and capabilities. A primary difference lies in what is supported in terms of covariance structures. In particular, `SAS PROC MIXED` and `mmrm` are the only implementations which provide support for many of the presented covariance structures. While `lme4::lmer` and `glmmTMB::glmmtmb` only have direct capabilities for unstructured covariance structures, `nlme::gls` provides a path through for unstructured, heterogeneous compound symmetry, and heterogeneous auto-regressive (ar1) structures. 
+
+# Data Sets
+
+The FEV data set is contains measurements of FEV1 (forced expired volume in one second), a measure of how quickly the lungs can be emptied. Low levels of FEV1 may indicate chronic obstructive pulmonary disease (COPD). This data set is used to illustrate model fitting with the `mmrm`, `lme4`, `nlme`, `glmmtmb` R packages as well as `SAS`.
+
+The FEV data set is summarized below.
+
+```
+                                      Stratified by ARMCD
+                               Overall       PBO           TRT           
+  n                              800           420           380        
+  USUBJID 
+     PT[1-200]                   800           420           380  
+  AVISIT (%)                                                             
+     VIS1                        200 (25.0)    105 (25.0)     95 ( 25.0) 
+     VIS2                        200 (25.0)    105 (25.0)     95 ( 25.0) 
+     VIS3                        200 (25.0)    105 (25.0)     95 ( 25.0) 
+     VIS4                        200 (25.0)    105 (25.0)     95 ( 25.0) 
+  ARMCD = TRT (%)                380 (47.5)      0 ( 0.0)    380 (100.0) 
+  RACE (%)                                                               
+     Asian                       280 (35.0)    152 (36.2)    128 ( 33.7) 
+     Black or African American   300 (37.5)    184 (43.8)    116 ( 30.5) 
+     White                       220 (27.5)     84 (20.0)    136 ( 35.8) 
+  SEX = Female (%)               424 (53.0)    220 (52.4)    204 ( 53.7) 
+  FEV1_BL (mean (SD))          40.19 (9.12)  40.46 (8.84)  39.90 (9.42)  
+  FEV1 (mean (SD))             42.30 (9.32)  40.24 (8.67)  44.45 (9.51)  
+  WEIGHT (mean (SD))            0.52 (0.23)   0.52 (0.23)   0.51 (0.23)  
+  VISITN (mean (SD))            2.50 (1.12)   2.50 (1.12)   2.50 (1.12)  
+  VISITN2 (mean (SD))          -0.02 (1.03)   0.01 (1.07)  -0.04 (0.98)  
+```
+
+# Model implementations {.tabset}
```

Comment:
Might be good to talk about degrees of freedom here. `mmrm` aims to have them all, `nmle::gls` and `lme4::lmer` do not support KR out of the box only via `pbkrtest`, right?

Reply:
we can also source from https://github.com/RConsortium/asa-biop-swe-wg/blob/main/slides/r-adoption-jan2023.qmd and other presentations in that folder